sdks/java/core/src/test/java/org/apache/beam/sdk/schemas/transforms/SelectTest.java

```diff
@@ -594,6 +595,64 @@ public void testFlatSchema() {
     pipeline.run();
   }
 
+  @Test
+  @Category(NeedsRunner.class)
+  public void testFlatSchemaWithArrayNestedField() {
+
```

Comment:
Let's add a description for this test, I suppose, this way we can provide a much better understanding to interested people. E.g. describe nested schema and expected flattened version of this schema.

Reply:
Done